packages/zipkin/index.d.ts

```diff
@@ -61,6 +62,11 @@ declare namespace zipkin {
     recordLocalAddr(inetAddress: InetAddress): void;
     recordBinary(key: string, value: boolean | string | number): void;
     writeIdToConsole(message: any): void;
+    /** Extract propagation ctx from request */
+    extractId(readHeader: <T> (header: string) => option.IOption<T>): void;
+    /** Injector propagation ctx from request */
+    injector(request: any): object;
```

Comment:
injector can be for messaging too so I think it is better that we pass something like `readHeader` and call it `setter` and this return an object that performs the injection, does that make sense?